docs/conf.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+#
+# lstchain documentation build configuration file, created by
+# sphinx-quickstart on Tue Aug 11 14:07:30 2020.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+import lstchain
+import os
+import datetime
+
+# Get configuration information from setup.cfg
+from configparser import ConfigParser
+
+conf = ConfigParser()
+conf.read([os.path.join(os.path.dirname(__file__), "..", "setup.cfg")])
+setup_cfg = dict(conf.items("metadata"))
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+needs_sphinx = '1.5'
```

Comment:
We require 3.5 in `setup.py`.

Can you test if it works with the current release? 4.2? We had some issues with it in ctapipe I was not yet able to resolve, but maybe this does not affect lstchain.

Reply:
I tried with the latest 4.3 and it worked.

Reply:
that line was directly copied from:
https://github.com/cta-observatory/ctapipe/blob/a2d5d084c0b9fba68646b09cfd20ae3f733fff4e/docs/conf.py#L39

maybe it needs to be updated there as well